src/types/flows.ts

```diff
@@ -119,6 +119,8 @@ export interface Flow {
   meta: ResourceManipulator<PipeMeta>
   results: FluxResult
   readOnly?: boolean
+  createdAt?: Date
+  updatedAt?: Date
```

Comment:
not sure if this should be optional or if this should be required? Like, when we create a flow that data doesn't exist so it seems like it should be optional but 🤷🏽

Reply:
updated should totes be updated... and thinking about how many times we have leaned on the offline first mechanism.... i feel like the created date should also be optional to communicate "this has been first saved to the system"